bot/exts/help_channels/_cog.py

```diff
@@ -332,6 +354,10 @@ async def move_to_available(self) -> None:
             category_id=constants.Categories.help_available,
         )
 
+        # Adding the help channel to the dynamic message, and editing/sending that message.
+        self.available_help_channels.add(channel.id)
+        await self.update_available_help_channels()
```

Comment:
You can save one edit if you remove this update. The next thing that runs after this task is another update.